readthedocs/doc_builder/environments.py

```diff
@@ -457,7 +457,8 @@ def run_command_class(
             if warn_only:
                 msg = 'Command {cmd} failed'.format(cmd=build_cmd.get_command())
                 if build_cmd.output:
-                    msg += ':\n{out}'.format(out=build_cmd.output)
+                    truncated_output = "\n".join(build_cmd.output.split("\n")[:10])
+                    msg += ":\n{out}".format(out=truncated_output)
                 log.warning(
                     msg,
```

Comment:
Probably we should set the msg to "Command failed" and log the command and the output as kwargs of `log.warning`, this is so we can filter all these easily

Reply:
I like this idea too.